exporter/logzioexporter/factory_test.go

```diff
@@ -50,9 +50,9 @@ func TestGenerateUrl(t *testing.T) {
 	generateURLTests := []generateURLTest{
 		{"", "us", "https://listener.logz.io:8071/?token=token"},
 		{"", "", "https://listener.logz.io:8071/?token=token"},
-		{"https://doesnotexist.com", "", "https://doesnotexist.com"},
-		{"https://doesnotexist.com", "us", "https://doesnotexist.com"},
-		{"https://doesnotexist.com", "not-valid", "https://doesnotexist.com"},
+		{"https://nonexistent.com", "", "https://nonexistent.com"},
```

Comment:
I sometimes encourage people to use RFC 2606 domains in which case this should be https://nonexistent.example.com, but I skipped that step here...

Reply:
I agree. Maybe good to track with an other issue.